src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorDocumentSynchronizationEndpoint.cs

```diff
@@ -80,38 +80,33 @@ public async Task<Unit> Handle(DidChangeTextDocumentParams notification, Cancell
             var sourceText = await document.GetTextAsync();
             sourceText = ApplyContentChanges(notification.ContentChanges, sourceText);
 
-            var version = GetDocumentVersion(notification.TextDocument.Version);
+            if(notification.TextDocument.Version is null)
+            {
+                throw new NotImplementedException("Provided version should not be null.");
```

Comment:
`InvalidOperationException` is a better approach here. Ditto with the other NotImplemented exception below